packages/react-native-client/src/hooks/useCamera.ts

```diff
@@ -45,41 +46,96 @@ export type CameraConfig<MetadataType extends Metadata> = {
    * whether to flip the dimensions of the video, that is whether to film in vertical orientation.
    * @default `true`
    */
-  flipVideo?: boolean;
+  flipVideo?: boolean /**
+   * whether the camera track is initially enabled, you can toggle it on/off later with toggleCamera method
+   * @default `true`
+   */;
+  cameraEnabled?: boolean;
   /**
-   * a map `string -> any` containing video track metadata to be sent to the server.
+   * id of the camera to start capture with. Get available cameras with `getCaptureDevices()`.
+   * You can switch the cameras later with `flipCamera`/`switchCamera` functions.
+   * @default the first front camera
    */
-  videoTrackMetadata?: MetadataType;
+  captureDeviceId?: string;
+};
+
+export type CameraConfig = CameraConfigBase & {
   /**
    *  SimulcastConfig of a video track. By default simulcast is disabled.
    */
-  simulcastConfig?: SimulcastConfig;
+  simulcastEnabled?: boolean;
```

Comment:
Is comment above outdated or should it be updated?

Reply:
I missed that comment has different prop name. Anyway, I've updated it!